packages/nimble-components/src/wafer-map/tests/wafer-map.stories.ts

```diff
@@ -17,17 +17,67 @@ import {
 } from './sets';
 
 interface WaferMapArgs {
-    colorscale: WaferMapColorsScale;
-    colorscalemode: WaferMapColorsScaleMode;
-    dielabelshidden: boolean;
-    dielabelsuffix: string;
+    colorsScale: WaferMapColorsScale;
+    colorsScaleMode: WaferMapColorsScaleMode;
+    dielabelsHidden: boolean;
```

Comment:
These variable names should be `dieLabelsHidden` and `dieLabelsSuffix` (capital L on Labels since it's a separate word).

Reply:
Fixed